runtime/parachains/src/inclusion.rs

```diff
@@ -262,24 +261,36 @@ impl<T: Trait> Module<T> {
 		let threshold = availability_threshold(validators.len());
 
 		let mut freed_cores = Vec::with_capacity(n_bits);
-		for (para_id, pending_availability, commitments) in assigned_paras_record.into_iter()
+		for (para_id, pending_availability) in assigned_paras_record.into_iter()
 			.filter_map(|x| x)
-			.filter_map(|(id, p, c)| p.and_then(|p| c.map(|c| (id, p, c))))
+			.filter_map(|(id, p)| p.map(|p| (id, p)))
 		{
 			if pending_availability.availability_votes.count_ones() >= threshold {
 				<PendingAvailability<T>>::remove(&para_id);
-				<PendingAvailabilityCommitments>::remove(&para_id);
+				let commitments = match <PendingAvailabilityCommitments>::take(&para_id) {
+					Some(commitments) => commitments,
+					None => {
+						debug::warn!(r#"
+						Inclusion::process_bitfields:
+							PendingAvailability and PendingAvailabilityCommitments
+							are out of sync, did someone messed up with the storage?
```

Comment:
```suggestion
							are out of sync, did someone mess with the storage?
```

I like the `debug::warn!`. Just tweaking grammar ("mess with", "did someone mess up the storage" would both be correct)